examples/dt/dcn/service-values.yaml

```diff
@@ -196,26 +196,62 @@ data:
         replicas: 1
         type: edge
   manila:
-    enabled: false
+    enabled: true
     manilaAPI:
       customServiceConfig: |
         [DEFAULT]
         enabled_share_protocols=nfs,cephfs
-    manilaShares:
-      share1:
-        customServiceConfig: |
-          [DEFAULT]
-          enabled_share_backends = cephfs
-          enabled_share_protocols = cephfs
-          [cephfs]
-          driver_handles_share_servers = False
-          share_backend_name = cephfs
-          share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
-          cephfs_conf_path = /etc/ceph/ceph.conf
-          cephfs_cluster_name = ceph
-          cephfs_auth_id=openstack
-          cephfs_volume_mode = 0755
-          cephfs_protocol_helper_type = CEPHFS
+    az0:
```

Comment:
this should be:
```
manilaShares:
  az0:
    customServiceConfig:
    ...
  az1:
    customServiceConfig:
    ...
  az2:
    customServiceConfig:
    ...
```

Reply:
Thanks, done!

Reply:
I might be wrong, but `manilaShares` has a wrong indentation, and is not aligned w/ `manilaAPI` and `manilaScheduler`.

Reply:
Done